pkg/types/installconfig.go

```diff
@@ -11,10 +11,20 @@ import (
 const (
```

Comment:
can we allow the platform specify package to define its name?

Reply:
sure i think that would be better.

Reply:
> can we allow the platform specify package to define its name?

Done with 9bf2fe6 -> 6a04f63.